Microsoft.Azure.CosmosRepository/src/Dtos/PagedResult.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.Azure.CosmosRepository.Dtos
+{
+    /// <inheritdoc />
+    public class PagedResult<T> : ListResult<T>
+    {
+        /// <inheritdoc />
+        public long TotalCount { get; set; }
+
+        /// <summary>
+        /// Creates a new <see cref="PagedResult{T}"/> object.
+        /// </summary>
+        public PagedResult()
+        {
+
+        }
```

Comment:
```suggestion
        /// <summary>
        /// Creates a new <see cref="PagedResult{T}"/> object.
        /// </summary>
        public PagedResult() { }
```

Nit: for empty `.ctor` inline declaration to avoid multiple lines.